acestep/acestep_v15_pipeline.py

```
@@ -236,7 +236,7 @@ def main():
                 code
                 + f" ({native_name}"
                 + (f"/{name})" if name != native_name else ")")
-                for code, name, native_name in available_languages
+                for code, name, native_name, dir in available_languages
```

Ruff (0.15.12)

[error] Variable `dir` is shadowing a Python builtin

(A001)